api/inventorypb/agents.proto

```diff
@@ -52,6 +52,8 @@ message NodeExporter {
   bool disabled = 3;
   // Custom user-assigned labels.
   map<string, string> custom_labels = 4;
+  // True if exporter use pull metrics mode.
+  bool push_metrics_disabled = 7;
```

Comment:
so maybe it's better to call it `pull_metrics_enabled` or `push_metrics_enabled` then?
or even better `metrics_mode` since we already have enum for it.
@AlekSi what do you think?

Reply:
Enum exists only in Management API because only in it we have "auto". In Inventory API and database layer we have a single boolean toggle. See ticket description

Reply:
We should keep previous behaviour, so for existed agents we should return field with `false` value. that's why I'm suggesting to use `push_metrics_enabled`

Reply:
done